### PR TITLE
Added state list to service.

### PR DESCRIPTION
**Pull request by ocampd1**

### CODE REVIEW

File: bp-web-client/angular-app/src/app/components/app-contact/app-contact.component.ts [hunk 1]
```diff
@@ -27,7 +28,8 @@ export class ContactComponent implements OnInit {
   constructor(private quoteService: QuoteService,
               private spinner: NgxSpinnerService,
               private appAlertOverlayModalService: AppAlertOverlayModalService,
-              public scrollTo: ScrollService) {
+              public scrollTo: ScrollService,
+              public list: StateListService) {
```
Comment (pouncilt) [inline bp-web-client/angular-app/src/app/components/app-contact/app-contact.component.ts:32]: Make constructor variables private.  Rename `list` variable to `stateNameService`.

File: bp-web-client/angular-app/src/app/components/app-contact/app-contact.component.ts [hunk 2]
```diff
@@ -27,7 +28,8 @@ export class ContactComponent implements OnInit {
   constructor(private quoteService: QuoteService,
               private spinner: NgxSpinnerService,
               private appAlertOverlayModalService: AppAlertOverlayModalService,
-              public scrollTo: ScrollService) {
+              public scrollTo: ScrollService,
```
Comment (pouncilt) [inline bp-web-client/angular-app/src/app/components/app-contact/app-contact.component.ts:31]: Make constructor variables private.  Rename `scrollTo` variable to `scrollToService`.

File: bp-web-client/angular-app/src/app/services/state-list.service.ts [hunk 3]
```diff
@@ -0,0 +1,62 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class StateListService {
+  states: any = [
```
Comment (pouncilt) [inline bp-web-client/angular-app/src/app/services/state-list.service.ts:7]: Make public.

**Review by pouncilt — CHANGES_REQUESTED** [review]

File: bp-web-client/angular-app/src/app/components/app-contact/app-contact.component.ts (hunk 1 above)
Comment (ocampd1) [reply bp-web-client/angular-app/src/app/components/app-contact/app-contact.component.ts:32]: Renamed to stateNameService.

File: bp-web-client/angular-app/src/app/components/app-contact/app-contact.component.ts (hunk 2 above)
Comment (ocampd1) [reply bp-web-client/angular-app/src/app/components/app-contact/app-contact.component.ts:31]: renamed to scrollToService and adjusted code for it.

File: bp-web-client/angular-app/src/app/services/state-list.service.ts (hunk 3 above)
Comment (ocampd1) [reply bp-web-client/angular-app/src/app/services/state-list.service.ts:7]: Made var public.

**Review by pouncilt — APPROVED** [review]

**Pull request merged**